pkg/jx/cmd/common_install_test.go

```diff
@@ -14,8 +16,13 @@ func TestInstallEksctl(t *testing.T) {
 	defer os.Setenv("PATH", oldPath)
 
 	defer os.Unsetenv("JX_HOME")
-	err = os.Setenv("JX_HOME", "/tmp/"+uuid.New())
+	tempDir, err := ioutil.TempDir("", "common_install_test")
```

Comment:
`assert.NoError(t, err)`  or `assert.Nil(t, err)`?

Reply:
both seem to do the same thing in this case. @jenkins-x/core - what is the best style for this case?

Reply:
Looks like the difference is in the response message if the assertion fails: https://github.com/stretchr/testify/blob/master/assert/assertions.go#L1143